src/core/identity.h

```diff
@@ -213,6 +213,13 @@ class IdentityEx {
   IdentityEx(
       const IdentityEx& other);
 
+  IdentityEx( IdentityEx && other){
+      m_Verifier=std::move(other.m_Verifier);
+      m_ExtendedLen=other.m_ExtendedLen;
+      m_ExtendedBuffer=std::move(other.m_ExtendedBuffer);
+
+  }
+
```

Comment:
looks like there are some issues with adding explicit to the constructor that takes one argument.

```
/home/ubuntu/kovri/src/core/garlic.cc: In member function ‘void i2p::garlic::GarlicDestination::AddSessionKey(const uint8_t*, const uint8_t*)’:
/home/ubuntu/kovri/src/core/garlic.cc:396:27: error: no matching function for call to ‘i2p::crypto::CBCDecryption::SetKey(const uint8_t*&)’
     decryption->SetKey(key);

```

I will take a look at fixing that as well unless you tell me otherwise.  I did some research and it sounds like adding explicit will help avoid bugs.